doc/api/http.md

```diff
@@ -1963,6 +1888,7 @@ const req = http.request(options, (res) => {
 [`net.Server.listen(path)`]: net.html#net_server_listen_path_backlog_callback
 [`net.Server.listen(port)`]: net.html#net_server_listen_port_host_backlog_callback
 [`net.Server`]: net.html#net_class_net_server
+[`server.listen()`]: net.html#net_server_listen
```

Comment:
All of these should be in the sorted order.

Reply:
I am not certain if there is a lint rule for things like this but this would be pretty neat!

Reply:
I would like to address this in another PR and write some automation / linting to check for the order of the documented methods as well as the order of the links